### ISSUE TITLE
Filter out Member users on this tenant

### ISSUE THREAD

**Issue by AussieDavo** [Yvand/AzureCP#44]
Hi Yvand,

I have multiple authentication providers setup in SharePoint 2016. On-prem Active Directory for our internal users and Azure AD for Guest users.

I have Azure AD Connect synchronising all On-prem Active Directory accounts to our Office 365 Tenancy and noticed you have a setting in AzureCP to "Filter out Guest users on this tenant".

Would it be possible to put in a feature request to "Filter out Member users on this tenant" as we only really require AzureCP to lookup Guest users in our Tenancy.

**Comment by Yvand** [comment]
Hi @AussieDavo, thank you for your feedback, I will investigate this and report to this thread if I implement it.

**Comment by Yvand** [comment]
Hi @AussieDavo I implemented this feature and it will be in the next version

**Comment by Yvand** [comment]
@AussieDavo The release is now published: https://github.com/Yvand/AzureCP/releases/tag/14.0.20190307.660

**Comment by AussieDavo** [comment]
Thanks @Yvand I updated to 14.0.20190307.660 last weekend and can confirm that this works well. I am now able to only see Guest user accounts from AzureAD in People Picker which is exactly what I was looking for.

I did take me a few minutes to realise that the config was only available using PowerShell though. I removed my Tenant config and set it up again including the ExcludeMemberUsers option set to True:

`Add-Type -AssemblyName "AzureCP, Version=1.0.0.0, Culture=neutral, PublicKeyToken=65dc6b5903b51636"
$config = [azurecp.AzureCPConfig]::GetConfiguration("AzureCPConfig")

$newAADTenant = New-Object azurecp.AzureTenant
$newAADTenant.TenantName = "xxx.onMicrosoft.com"
$newAADTenant.ClientId = "Application ID"
$newAADTenant.ClientSecret = "XXX"
$newAADTenant.ExcludeMemberUsers = "True"
$config.AzureTenants.Add($newAADTenant)
$config.Update()`

Interestingly, in the GUI it shows "Filter out Guest" users as 'True", even if only the the ExcludeMemberUsers is set to True:

![image](https://user-images.githubusercontent.com/5260349/57340223-37effe00-7178-11e9-9897-df55fe151d6e.png)
